src/wodan/wodan.ml

```diff
@@ -1438,6 +1439,10 @@ struct
             else check_live_integrity fs child_alloc_id (Int64.succ depth) )
           entry.children_alloc_ids;
         if !fail then failwith "Integrity errors"
+[@@@warning "+32"]
+
+  (* This is equivalent to commenting out the above, while still having it typecheck *)
+  let check_live_integrity _ _ _ = ()
```

Comment:
There are two alternatives that are IMO slightly better:
-  `let[@@warning "-32"]`
-  `if false then` (or even better, `if do_check_live_integrity` with this variable set to `false` globally)

Reply:
1. Thanks!
2. I think I'd go with if false, just to let the compiler get rid of it easily